subtrack.MAUI/Pages/CreateSubscription.razor

```diff
@@ -1,13 +1,36 @@
 ﻿@page "/CreateSubscription"
 @inject NavigationManager NavigationManager
 
-<h3>CreateSubscription</h3>
-<br/>
-<p>returnUrl: @ReturnUrl</p>
+<div style="display: flex; justify-content: space-between; padding-bottom: 20px;">
+    <button type="button" class="btn btn-outline-dark">Back</button>
+    <button type="button" class="btn btn-outline-dark">Save</button>
```

Comment:
Keep the outline but make it use the  "primary" (blue) color

Reply:
I just committed without seeing these comments, sorry! 
![image](https://github.com/Code2Gether-Discord/subtrack/assets/92475149/0c68f8ac-4850-475d-9505-7aa8f45e8552)
I'll do as you've said in a bit.

Reply:
Nice with some btns instead, try making them a little smaller

Reply:
I'm actually having quite a bit of trouble with this.
`--bs-btn-padding-x: 2px; --bs-btn-padding-y: 2px;`
These are my two overrides from the btn in the css and I can't get it to work. 😭